ls1intum/Artemis
# [Feature] Always show failed buildlogs in result details

### Checklist

### Motivation and Context


In case we received any feedback, any build result would be ignored and not shown in the result details modal. 

### Description

In case we have received a failed build log, it is getting displayed together with the feedbacks. 

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create a programming exercise with the setups described below
4. Now create an SCA issue (e.g. in Java wrap code in redundant curly braces) and let the build fail (e.g. omitting a semicolon)
5. Verify that the result details shows the feedback and also the failed build logs
6. It should be shown in Online Code Editor, Course Exercise Detail, Course Exercise Row and Exams (anywhere where the result is accessible)

The following scenarios need to be tested (in total 8 programming exercise setups):
- Course + exam exercises
- With and without SCA enabled
- With and without online code editor

### Test Coverage



no changes.

### Screenshots


#### SCA Feedback + failed build logs:
![image](https://user-images.githubusercontent.com/33484318/110206749-f3227480-7e7f-11eb-8d93-8fec0936bf10.png)
